Sources/AppKitNavigation/Internal/AssociatedKeys.swift

```diff
@@ -0,0 +1,36 @@
+#if canImport(AppKit) && !targetEnvironment(macCatalyst)
+
+import AppKit
+
+struct AssociatedKeys {
```

Comment:
I couldn't find use of types defined in this file in this PR. Let's remove it till a PR needs them.